.github/workflows/testing.yml

```diff
@@ -88,8 +88,19 @@ jobs:
     - name: Install nox
       run: python -m pip install --progress-bar off --upgrade nox
 
-    - name: Install graphviz and pandoc
-      run: sudo apt install graphviz pandoc
+    - name: Install graphviz
+      run: sudo apt install graphviz
+
+    - name: Install pandoc
+      run: |
+        # Specify the desired version of Pandoc
+        PANDOC_VERSION="3.1.11.1" # Update this to the version you need
+        # Download the pre-built binary package for Linux
+        wget https://github.com/jgm/pandoc/releases/download/${PANDOC_VERSION}/pandoc-${PANDOC_VERSION}-1-amd64.deb
+        # Install the downloaded package
+        sudo dpkg -i pandoc-${PANDOC_VERSION}-1-amd64.deb
+        # Remove the downloaded package file
+        rm pandoc-${PANDOC_VERSION}-1-amd64.deb
```

Comment:
Hm...updating to the most recent release of pandoc still leads to the same error...